packages/theming/src/utils/getColor.ts

```diff
@@ -0,0 +1,225 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import { scale, valid } from 'chroma-js';
```

Comment:
While [chroma-js](https://gka.github.io/chroma.js/) is a wonderful library, it does add [15.7k](https://bundlephobia.com/package/chroma-js@2.4.2) to the bundle size. And because it's not tree-shakeable, even pulling in a couple methods will import the whole package.

In the future, we could attempt to reproduce something close to `scale` to reduce the JS footprint.

As for `valid`, we could potentially leverage [CSS.support](https://developer.mozilla.org/en-US/docs/Web/API/CSS/supports_static) (browser only). Ex: 

```
CSS.support('color', 'red') // true
CSS.support('color', '#fff') //true

CSS.support('color', 'bad') // false

```

Reply:
Great callout @ze-flo. Can I request that we allow this "brand backfill" feature of `getColor` through for now? I'd like to lean into the RFC process for evaluating efficacy of other approaches (let's take a closer look at [color2k](https://color2k.com/), etc). Specifically, let's add a `scale` research task to our v9 dev sequence. That way, we can get this into the hands of early adopters while continuing forward with the v9 effort.

Reply:
Absolutely! 💯